From eda45595e74228c043c32605289979ce5c369acd Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 29 Jan 2026 09:32:15 +0100
Subject: [PATCH 28/32] MdeModulePkg/CustomizedDisplayLib: fix gcc 16 warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c: In function ‘CreateDialog’:
MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c:435:18: error: variable ‘Count’ set but not used [-Werror=unused-but-set-variable=]
  435 |   UINTN          Count;
      |                  ^~~~~

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 .../Library/CustomizedDisplayLib/CustomizedDisplayLib.c       | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c b/MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c
index 1099b640a2d2..e7016845f22c 100644
--- a/MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c
+++ b/MdeModulePkg/Library/CustomizedDisplayLib/CustomizedDisplayLib.c
@@ -432,7 +432,6 @@ CreateDialog (
   UINTN          LargestString;
   UINTN          LineNum;
   UINTN          Index;
-  UINTN          Count;
   CHAR16         Character;
   UINTN          Start;
   UINTN          End;
@@ -503,9 +502,8 @@ CreateDialog (
   PrintCharAt ((UINTN)-1, (UINTN)-1, Character);
   Character = BOXDRAW_VERTICAL;
 
-  Count = 0;
   VA_START (Marker, Key);
-  for (Index = Top; Index + 2 < Bottom; Index++, Count++) {
+  for (Index = Top; Index + 2 < Bottom; Index++) {
     String = VA_ARG (Marker, CHAR16 *);
 
     if (String[0] == CHAR_NULL) {
-- 
2.52.0

